optimum/habana/transformers/models/glm4v/modeling_chatglm.py

```diff
@@ -1534,6 +1534,9 @@ def prepare_inputs_for_generation(
 
         if past_key_values:
```

Comment:
I don't understand this whole condition:  How checking past_key_values can assure if token_idx is not None?